README.md

```diff
@@ -15,6 +15,23 @@ To cite the Manubot project or for more information on its design and history, s
 Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br>
 _Manubot Preprint_ (2019) <https://greenelab.github.io/meta-review/>
 
+## Installation
+
+Install the latest release version:
+
+```sh
+pip install manubot
+```
+
+or install a version specified by a git commit hash:
+
+```sh
+COMMIT=33e512d21218263423de5f0d127aac4f8635468f
+pip install git+https://github.com/manubot/manubot@$COMMIT
+```
+
+Use the `--upgrade` argument to reinstall `manubot` with a different commit hash.
```

Comment:
I think it's possible that we could just add `--upgrade` to both `pip install` commands.

From https://github.com/conda/conda/pull/8542:

> Adding the flag here causes it to always be used, even for `conda env create`, but in that case the -U has no effect because the package is not already installed.

This way the commands work as intended regardless of whether `manubot` is already installed. We could add a comment or note that `--upgrade` ensures existing installations are replaced with the specified version as neccessary.